habitat-lab/requirements.txt

```diff
@@ -8,6 +8,7 @@ pickle5; python_version < '3.8'
 # visualization optional dependencies
 imageio>=2.2.0
 imageio-ffmpeg>=0.2.0
+pre-commit
```

Comment:
No reason this should be in requirements?

Reply:
Yeah my bad, I was looking up how to install pre-commit, and the website I was looking at said to add it to the requirements.txt file. Wasn't exactly sure if it was necessary